docs/markdown/DevGuide_InviwoPy.md

```diff
@@ -0,0 +1,72 @@
+# Using Inviwo from within Python
+While Python Processors (TODO: link) allow you to embed Python code inside Inviwo processors, you can also use Inviwopy to use your Inviwo networks from within a Python application. This guide shows you first how to build Inviwopy, then how to load existing Inviwo workspaces for use in your Python scripts and lastly some ways to modify a loaded Inviwo network from within Python.
+
+## Building InviwoPy
+To use Inviwopy, you have to build the appropriate `.dll`/`.so` yourself, since it is currently not available through Pypi. To do so, enable the `IVW_MODULE_PYTHON3` and `IVW_MODULE_PYTHON3QT` CMake flags. Next you need to specify the Python executable to which the produced library shall be compatible in the `PYTHON_EXECUTABLE` flag.
```

Comment:
there is nothing special here, this is exactly the same as when we build the regular python stuff...

Reply:
I agree. I would still leave it here and rather make sure it is the same in the other python guide as well. Or what changes to this part do you suggest?